Build fails on node versions below 11

**Describe the bug**
Following the build description in the README file leads to errors when using older versions of node (below 11), due to `worker_threads` being unavailable.

While not a bug in Viv per se, it might be good to document the necessity for node v11 or higher in the README file.

**To Reproduce**
Using a node version below 11:
```
$ npm run build

> @hms-dbmi/viv@0.10.5 build /home/claesenm/IdeaProjects/viv
> vite build --mode lib && tsc --emitDeclarationOnly

internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'worker_threads'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/claesenm/IdeaProjects/viv/node_modules/vite/dist/node/chunks/dep-cc49d7be.js:26:20)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
```

**Expected behavior**
A working build.

**Environment:**
Encountered issue with node version `v10.19.0`. Resolved after upgrading to `v14.17.5`.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,7 +8,7 @@
 
 - Upgrade deck.gl to 8.5
 - Fix changelog check in `bash`.
-
+- Update README to note Node version restrictions.
 ## 0.10.5
 
 ### Added

--- a/README.md
+++ b/README.md
@@ -84,6 +84,8 @@ Please install the [Prettier plug-in](https://prettier.io/docs/en/editors.html) 
 To run unit and integration tests locally, use `npm test`. For full prodcution test (including linting and formatting checks),
 use `npm run test:prod`.
 
+To our knowledge, Viv can be developed with Node version greater than 10.  To be certain of having a version that will work, you can check what is running on our CI by going to our [Github Workflow](https://github.com/hms-dbmi/viv/blob/master/.github/workflows/test.yml#L31).
+
 ## Build
 
 - `@hms-dbmi/viv` library: `npm run build`

